docs/installation/observability/metrics.rst

```diff
@@ -0,0 +1,214 @@
+.. _installation_observability_metrics:
+
+=======
+Metrics
+=======
+
+Open Klant produces application metrics (using Open Telemetry).
+
+.. note:: The exact metric names that show up may be transformed, e.g. Prometheus replaces
+   periods with underscores, and processing pipelines may add prefixes or suffixes.
+
+.. important:: Some metrics are defined as "global scope".
+
+   These metrics are typically derived from application state introspection, e.g. by
+   performing database (read) queries to aggregate some information. Usually those
+   correspond to an `Asynchronous Gauge <https://opentelemetry.io/docs/specs/otel/metrics/api/#asynchronous-gauge>`_.
+
+   Multiple replicas and/or instances of the same service will produce the same values
+   of the metrics. You need to apply some kind of aggregation to de-duplicate these
+   values. The attribute ``scope="global"``  acts as a marker for these type of metrics.
+
+   With PromQL for example, you can use ``avg`` on the assumption that all values will
+   be equal, so the average will also be identical:
+
+   .. code-block:: promql
+
+       avg by (type) (otel_user_count{scope="global"})
+
+Generic
+=======
+
+``http.server.duration``
+    Captures how long each HTTP request took, in ms. The metric produces histogram data.
+
+``http.server.request.duration`` (not active)
+    The future replacement of ``http.server.duration``, in seconds. Currently not
+    enabled, but the code is in the Open Telemetry SDK instrumentation already and could
+    possibly be opted-in to.
+
+Application specific
+====================
+
+Accounts
+--------
+
+``user_count``
+    Reports the number of users in the database. This is a global metric, you must take
+    care in de-duplicating results. Additional attributes are:
+
+    - ``scope`` - fixed, set to ``global`` to enable de-duplication.
+    - ``type`` - the user type. ``all``, ``staff`` or ``superuser``.
+
+    Sample PromQL query:
+
+    .. code-block:: promql
+
+        max by (type) (last_over_time(
+          otel_user_count{scope="global"}
+          [1m]
+        ))
+
+``auth.login_failures``
```

Comment:
similarly here - all the metrics should be prefixed with the `openklant` namespace. There's a PR open for OF that does this too: https://github.com/open-formulieren/open-forms/pull/5700